presto-main/src/main/resources/webapp/old/units.js

```diff
@@ -16,8 +16,8 @@ function formatDuration(millis) {
     return precisionRound(millis) + " " + unit;
 }
 
-function formatDataSize(bytes) {
-    var unit = "B";
+function formatDataSize(bytes, defaultUnit = "B") {
```

Comment:
How is this used? What if `defaultUnit != "B"` and `bytes < 1024`. What is example output?